### DISCUSSION TITLE
cannot use group-hover in tailwind v3

### DISCUSSION THREAD

**Discussion by ahsan2882** [tailwindlabs/tailwindcss#7157] [Help]
**What version of Tailwind CSS are you using?**
v3

**What build tool (or framework if it abstracts the build tool) are you using?**

React v17

**What version of Node.js are you using?**

14

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows

**Describe your issue**

Unable to call or use 'group' as a class in React, I have also tried in HTML. I am only able to call group-hover but it does not work since group cannot be defined.

**Comment by adamwathan** [comment]
Hey! Can you share a reproduction? I am building a React site as we speak and using `group-hover` so it must be something specific to your setup.

**Comment by leo-paz** [comment]
Hi I'm experience the same issue using NextJS and testing on chrome.

I have added these variants one by one and none seem to work 
![image](https://user-images.githubusercontent.com/26093865/151710853-b0500699-a079-4c57-9445-b0a58f32b64e.png)

I apply the group class like so
![image](https://user-images.githubusercontent.com/26093865/151710871-f460bce4-bdb2-4f6f-8374-601d58d9cb10.png)

And use group-hover like so and also tried with fill-current and changing the text color on group-hover
![image](https://user-images.githubusercontent.com/26093865/151710883-cb172f99-44b6-4c98-aff5-c702234b0dbb.png)

But it doesnt work at all? Am i missing something?
